utils/__init__.py

```diff
@@ -0,0 +1,8 @@
+import codecs
+
+def get_local_file_content(file_name):
+    """Read the contents of `file_name` into a unicode string, return the unicode string."""
+    f = codecs.open(file_name, encoding='utf-8')
+    ret = f.read().strip()
+    f.close()
```

Comment:
I think so. Though now that I look at it, this should likely be using a `try`/`except` to catch missing/unreadable files.